server/sonar-db-dao/src/main/java/org/sonar/db/DBSessionsImpl.java

```diff
@@ -69,7 +69,7 @@ private static void ensureAutoCommitFalse(DbSession dbSession) {
         checkState(!((boolean) f.get(sqlSession)), "Autocommit must be false");
       }
     } catch (NoSuchFieldException | IllegalAccessException e) {
-      LOG.debug("Failed to check the autocommit status of SqlSession: {}", e);
+      LOG.debug("Failed to check the autocommit status of SqlSession: {}", e.getMessage());
```

Comment:
```suggestion
      LOG.debug("Failed to check the autocommit status of SqlSession: {}", e.getMessage(), e);
```

Reply:
I did it, 🤝.
Thank you for the feedback @pierre-guillot-gh!